internal/database/release.go

```diff
@@ -599,37 +607,68 @@ func (repo *ReleaseRepo) Delete(ctx context.Context, req *domain.DeleteReleaseRe
 		}
 	}
 
+	//if req.OlderThan > 0 {
+	//	thresholdTime := time.Now().Add(-time.Duration(req.OlderThan) * time.Hour)
+	//	qb = qb.Where("timestamp < ?", thresholdTime)
+	//}
+
+	if len(req.Indexers) > 0 {
+		placeholders := strings.Repeat("?,", len(req.Indexers)-1) + "?"
+		qb = qb.Where(fmt.Sprintf("indexer IN (%s)", placeholders), toInterfaceSlice(req.Indexers)...)
```

Comment:
There is a built in `sq.Placeholders(len(req.Indexers))` function you can use here.

```diff
-		placeholders := strings.Repeat("?,", len(req.Indexers)-1) + "?"
-		qb = qb.Where(fmt.Sprintf("indexer IN (%s)", placeholders), toInterfaceSlice(req.Indexers)...)
+		qb = qb.Where(fmt.Sprintf("indexer IN (%s)", sq.Placeholders(len(req.Indexers))), toInterfaceSlice(req.Indexers)...)
```

And it maybe should be using `sq.Expr` instead of a fmt.Sprintf 🤔 or just a `sq.Eq` per https://github.com/Masterminds/squirrel/issues/334 should work.